components/steering_wheel/src/IO/IO_Digital.c

```diff
@@ -0,0 +1,73 @@
+/**
+ * IO_Digital.c
+ * Digital IO Module source code
+ * @note To transfer this module to another system, the initialized inputs
+ *      of IO_DIGITAL must be modified to be congruent to the new system
+ */
+
+/******************************************************************************
+ *                             I N C L U D E S
+ ******************************************************************************/
+
+// Module Header
+#include "IO/IO_Digital.h"
+
+// System includes
+#include <string.h>
+
+// Other includes
+#include "ModuleDesc.h"
+#include "SystemConfig.h"
+#include "Utility.h"
+
+
+/******************************************************************************
+ *                         P R I V A T E  V A R S
+ ******************************************************************************/
+
+const HW_GPIO_Input_S inputs[NUM_INPUTS] = {
```

Comment:
nit: formatting. With a table like this it's always nice to have the columns aligned so everything is clear